Makefile

```diff
@@ -75,17 +75,17 @@ test-vectors: pull-serialization-tests run-vectors
 test:
 	cargo test --all --exclude serialization_tests --exclude forest_message --exclude forest_crypto -- --test-threads=$(RUST_TEST_THREADS)
 	cargo test -p forest_crypto --features blst --no-default-features -- --test-threads=$(RUST_TEST_THREADS)
+	cargo test --all --exclude serialization_tests --exclude forest_message -- --test-threads=$(RUST_TEST_THREADS)
 	# FIXME: https://github.com/ChainSafe/forest/issues/1444
```

Comment:
should this comment go as well in this case?

Reply:
Yea I messed up restoring the rule here, It should have been